maven-plugin/src/main/java/org/openmrs/maven/plugins/BuildDistro.java

```diff
@@ -208,11 +212,12 @@ private String buildDistro(File targetDirectory, Artifact distroArtifact, Distro
             try {
                 ZipFile warfile = new ZipFile(new File(web, OPENMRS_WAR));
                 File tempDir = new File(web, "WEB-INF");
+                tempDir.mkdir();
                 moduleInstaller.installModules(distroProperties.getModuleArtifacts(distroHelper, targetDirectory),
                         new File(tempDir, WAR_FILE_MODULES_DIRECTORY_NAME).getAbsolutePath());
 
                 File owasDir = new File(tempDir, "bundledOwas");
-                owasDir.mkdirs();
```

Comment:
Was the change from `mkdirs` to `mkdir` intentional?

Reply:
Yeah, when I did the refactor of the `setup` task I eventually realized that a `owasDir.mkdirs()` call was actually accidentally being used to create the server directory itself. This made it very hard to reason about what was happening when. That gave me the belief that `mkdirs` should be used sparingly. It opens the door for code to get messy and dependent on what should be side-effects.

Reply:
How about `web.mkdirs()` above?

Reply:
Since that's the "top" of the directory tree that `BuildDistro` is concerned with, I have less faith that its parent directories have been created already.